apps/notifications/src/app/note-list/index.tsx

```diff
@@ -48,6 +50,8 @@ const NoteList = ( { filterName }: { filterName: keyof ReturnType< typeof getFil
 	const { data: filteredData, paginationInfo } = filterSortAndPaginate( notes, view, fields );
 
 	const onChangeSelection = ( selection: string[] ) => {
+		const noteId = selection[ 0 ];
+		dispatch( actions.ui.selectNote( noteId ) );
```

Comment:
Hmm I think it makes more sense to call this when `apps/notifications/src/app/note/index.tsx` mounts (e.g. in useEffect)?

It will be useful when we go to a notification via keyboard shortcut, not from the note list (e.g. using `j` to go to the next notification).

Reply:
Hmm, what's weird about it 🤔 it's similar to `<PageViewTracker />` where we fire something on component mount.

Reply:
> Would it be better to dispatch the action via shortcut as well?

There are many other ways to reach a note, for example we can focus on it by tabbing and then click space / enter 🥲

Reply:
Like [Notifying parent components about state changes](https://react.dev/learn/you-might-not-need-an-effect#notifying-parent-components-about-state-changes), it would be better to do everything in a single pass.

> There are many other ways to reach a note, for example we can focus on it by tabbing and then click space / enter 🥲

🥲